packages/event_source_client/lib/src/sse_client_html.dart

```diff
@@ -49,6 +49,11 @@ class HtmlSseClient implements SSEClient {
 
   @override
   Future close() => _messageEventsController.close();
+
+  @override
+  void restart() {
+    // TODO: implement restart
```

Comment:
Not implementing this right now. We need to implement jitter/backoff for the web first.